docs/howto/manage-charms.md

```diff
@@ -84,20 +71,23 @@ To deploy a charm / bundle from [Charmhub](https://charmhub.io/) / your local fi
 juju deploy <charm | charm bundle> | <path to the local charm or bundle>
 ```
 
-````{dropdown} Example: Deploy a Charmhub charm
+The command also allows you to add another argument to specify a custom name (alias) for your deployed application (charms only). You can also take advantage of the rich set of flags to specify a charm channel or revision, a machine base, a machine constraint (e.g., availability zone), the number of application units you want (clusterised), a space binding, a placement directive (e.g., to deploy to a LXD container), a specific storage instance, a specific machine, etc., and even to trust the application with the current credential -- in case the application requires access to the backing cloud in order to fulfil its purpose (e.g., stojrage-related tasks).
```

Comment:
>  you to add another argument to specify a custom name (alias) for your deployed application (charms only).

You can do this for bundles in the bundle text. Reads a bit odd to me.

Reply:
Changed to

> to your deployed application(s).

Thanks!